crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/service/stateless/FrequencyAmount.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.justice.laa.crime.meansassessment.service.stateless;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotNull;
+import uk.gov.justice.laa.crime.meansassessment.staticdata.enums.Frequency;
+
+import java.math.BigDecimal;
+
+public class FrequencyAmount {
```

Comment:
Please see my other comment around the use of Lombok annotations to reduce boilerplate.